generators/2.0/java/templates/build.gradle.ejs

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java-library'
+    id 'application'
+}
+
+java {
```

Comment:
that could be up to the user to decide

Reply:
Can we let it empty?

Reply:
In that case you could even remove the block
(and the version used would probably the one from the JDK used to run Gradle)

Reply:
Okay, I removed the entire block